utils_test.go

```diff
@@ -279,6 +279,25 @@ func TestDecompression(t *testing.T) {
 	}
 }
 
+func TestGetSessionTimeout(t *testing.T) {
+	req, err := http.NewRequest(http.MethodPost, "http://127.0.0.1:9090", nil)
+	if err != nil {
+		panic(err)
+	}
+	params := make(url.Values)
+	params.Add("query", "SELECT 1")
+	params.Set("session_timeout", "888")
+	req.URL.RawQuery = params.Encode()
+	trueSessionTimeout := getSessionTimeout(req)
+	t.Logf("user set session_timeout true , return is %d",  trueSessionTimeout)
```

Comment:
if you only log, the test will pass even if the logic doesn't work inside getSessionTimeout().
So, if someone breaks this part, the automatic CI won't see it.

Can you rewrite the logic (for the 2 scenarios in your test) using a check like this one?
	sessionTimeout != [expected value] {
		t.Fatalf("[blabla] got: %q; expected %q", sessionTimeout, [expected value] )
	}